powersimdata/scenario/state.py

```diff
@@ -28,6 +29,19 @@ def refresh(self, scenario):
         """
         pass
 
+    def _update_scenario_info(self):
```

Comment:
I think I see what's going on here, but maybe it would be clearer if we add a comment that this method is overwritten in the Execute class, and that's why we have a placeholder here.

Reply:
I changed the docstring to be more useful, but didn't mention specific classes so if things change later the description won't also need to change.